az-digital/az_quickstart
# Fixes #2214 Add a View to Publications to have all searches in one View

## Description
This PR changes the publication views so that the the four views providing one exposed filter each are merged into a single view with four filters. This more correctly fits the intent of the original #2118 ticket and is more maintainable than four separate views.

The AZ Publications Person view that supplies a block for placing on the person page that shows a person's publications is unchanged as this is for a separate purpose than the views with exposed forms.

## Related issues
#2214 

## How to test
- Verify `az_publication` is in the package `The University of Arizona - Experimental`
- enable `az_publication`
- add `AZ Publications` view to a flexible page, selecting one of the citation styles as the display
- Add some publications and authors
- Visit the publication view page you created and verify the filters are working correctly
- verify that on desktop size all four filters are on a single line that lines up correctly
- verify the date input select box has bootstrap styles applied and is not an unstyled select element 

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
